pkg/standalone/stop.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 // Stop terminates the application process.
-func Stop(appID string) error {
+func Stop(appID string, cliPIDToNoOfApps map[int]int) error {
```

Comment:
This particular parameter is added to keep track of how many applications launched using the same `CLIPID`. This will be helpful in deciding when to kill the CLIPID.